### PR TITLE
Bl user profile

### PR DESCRIPTION
**Pull request by tluu5**

### CODE REVIEW

File: app/views/users/show.html.erb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<h1>
+  <%= @user.username%>
+</h1>
+
+<h2>Own photos</h2>
+
+<% @user.own_photos.each do |photo| %>
+  <div class="row mb-4">
+    <div class="col-md-6 offset-md-3">
```
Comment (jessjno) [inline app/views/users/show.html.erb:9]: good job not forgetting the first '-' in 'col-md-6'

File: app/views/users/show.html.erb
Comment (jessjno) [inline app/views/users/show.html.erb:1]: nice formatting

**Review by jessjno — APPROVED** [review]

File: app/controllers/users_controller.rb
Comment (dsimp) [inline app/controllers/users_controller.rb:1]: good strict find_by

**Review by dsimp — APPROVED** [review]

File: app/controllers/comments_controller.rb [hunk 2]
```diff
@@ -22,10 +22,11 @@ def edit
   # POST /comments or /comments.json
   def create
     @comment = Comment.new(comment_params)
+    @comment.author = current_user
 
     respond_to do |format|
       if @comment.save
-        format.html { redirect_to comment_url(@comment), notice: "Comment was successfully created." }
+        format.html { redirect_back fallback_location: root_path, notice: "Comment was successfully created." }
```
Comment (marywebby) [inline app/controllers/comments_controller.rb:29]: Good job implementing `redirect_back`

File: app/views/comments/_form.html.erb [hunk 3]
```diff
@@ -11,22 +11,13 @@
...
-  <div>
-    <%= form.label :body, style: "display: block" %>
-    <%= form.text_area :body %>
+  <div class="mb-3">
+    <%= form.text_area :body, class: "form-control" %>
   </div>
 
   <div>
-    <%= form.submit %>
+    <%= form.submit class: "btn btn-outline-secondary btn-block" %>
```
Comment (marywebby) [inline app/views/comments/_form.html.erb:21]: good refactoring of the form

File: app/views/users/show.html.erb [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+        </div>
+        <ul class="list-group list-group-flush">
+          <% photo.comments.each do |comment| %>
+            <li class="list-group-item">
+              <div class="d-flex">
+                <div class="flex-shrink-0">
+                  <img src="..." alt="...">
+                </div>
+                <div class="flex-grow-1 ms-3">
+                  <h5 class="mt-0"><%= comment.author.username %></h5>
```
Comment (marywebby) [inline app/views/users/show.html.erb:24]: nice job also including the author.username

File: config/routes.rb [hunk 5]
```diff
@@ -6,6 +6,7 @@
   resources :follow_requests
   resources :comments
   resources :photos
+  get "/:username" => "users#show", as: :user
```
Comment (marywebby) [inline config/routes.rb:9]: Make sure to include devise for users, I know you have it already, but it is meant to be inside this PR.

**Review by marywebby — APPROVED** [review]
Nice work!

**Pull request merged**